src/coffee/client/client.coffee

```diff
@@ -3,22 +3,36 @@
 # All rights reserved.
 #
 
-angular = require 'angular'
-
-############################################################################################################
-
 # Allow Node.js-style `global` in addition to `window`
 if typeof(global) is 'undefined'
     window.global = window
 
+# Load JSData libraries. The core library must be first.
+require 'js-data'
+window.DSLocalStorageAdapter = require 'js-data-localstorage'
```

Comment:
For something which looks odd (as this does), it's usually best to add a comment explaining why it has to be this way.  In actual production code, it's probable that some other developer will come along and "clean up" this strange piece of code and break things.  Leaving a comment will prevent that.